test/commands/remove/alias_spec.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require "./test/helper"
+
+clean_describe "remove alias" do
+  subject { run_cmd("remove alias #{location_name} #{nickname}") }
```

Comment:
Could you also add tests for when `location_name` and `nickname` are `nil`, and when just `nickname` is?